### PR TITLE
U4-9319 Update to latest ImageProcessor v2.5.0 and ImageProcessor.Web…

### PR DESCRIPTION
**Pull request by Shazwazza**
… v4.7.0

### CODE REVIEW

File: src/Umbraco.Core/Umbraco.Core.csproj [hunk 1]
```diff
@@ -52,8 +52,8 @@
     <Reference Include="ICSharpCode.SharpZipLib">
       <HintPath>..\packages\SharpZipLib.0.86.0\lib\20\ICSharpCode.SharpZipLib.dll</HintPath>
     </Reference>
-    <Reference Include="ImageProcessor, Version=2.4.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\ImageProcessor.2.4.5.0\lib\net45\ImageProcessor.dll</HintPath>
+    <Reference Include="ImageProcessor, Version=2.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\ImageProcessor.2.5.0.0\lib\net45\ImageProcessor.dll</HintPath>
```
Comment (emilwangaa) [inline src/Umbraco.Core/Umbraco.Core.csproj]: So this doesn't work for me. Nuget restores the package fine but it restores to this folder ImageProcessor.2.5.0 and not to the hintpath ImageProcessor.2.5.0.0. Can't really figure out why that is.

File: src/Umbraco.Web.UI/Umbraco.Web.UI.csproj [hunk 2]
```diff
@@ -135,12 +135,12 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\SharpZipLib.0.86.0\lib\20\ICSharpCode.SharpZipLib.dll</HintPath>
     </Reference>
-    <Reference Include="ImageProcessor, Version=2.4.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\ImageProcessor.2.4.5.0\lib\net45\ImageProcessor.dll</HintPath>
+    <Reference Include="ImageProcessor, Version=2.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\ImageProcessor.2.5.0.0\lib\net45\ImageProcessor.dll</HintPath>
```
Comment (emilwangaa) [inline src/Umbraco.Web.UI/Umbraco.Web.UI.csproj]: Same problem as above.

File: src/Umbraco.Web.UI/Umbraco.Web.UI.csproj [hunk 3]
```diff
@@ -135,12 +135,12 @@
...
-    <Reference Include="ImageProcessor, Version=2.4.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\ImageProcessor.2.4.5.0\lib\net45\ImageProcessor.dll</HintPath>
+    <Reference Include="ImageProcessor, Version=2.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\ImageProcessor.2.5.0.0\lib\net45\ImageProcessor.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ImageProcessor.Web, Version=4.6.6.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\ImageProcessor.Web.4.6.6.0\lib\net45\ImageProcessor.Web.dll</HintPath>
+    <Reference Include="ImageProcessor.Web, Version=4.7.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\ImageProcessor.Web.4.7.0.0\lib\net45\ImageProcessor.Web.dll</HintPath>
```
Comment (emilwangaa) [inline src/Umbraco.Web.UI/Umbraco.Web.UI.csproj]: And it also applies for the ImageProcessor.Web. Restores to ImageProcessor.Web.4.7.0 and not ImageProcessor.Web.4.7.0.0

File: src/Umbraco.Core/Umbraco.Core.csproj (hunk 1 above)
Comment (zpqrtbnk) [inline src/Umbraco.Core/Umbraco.Core.csproj]: Can confirm the problem (figured it out while looking at what was causing my build light to be red)

**Pull request merged**